src/tests/krate.rs

```diff
@@ -1589,6 +1589,120 @@ fn publish_after_yank_max_version() {
     assert_eq!(json.krate.max_version, "2.0.0");
 }
 
+#[test]
+fn publish_records_an_audit_action() {
+    use cargo_registry::models::{VersionAction, VersionOwnerAction};
+
+    let (app, anon, _, token) = TestApp::full().with_token();
+
+    app.db(|conn| assert!(VersionOwnerAction::all(&conn).unwrap().is_empty()));
+
+    // Upload a new crate, putting it in the git index
+    let crate_to_publish = PublishBuilder::new("fyk");
+    token.enqueue_publish(crate_to_publish).good();
+
+    // make sure it doesn't have any audit actions
```

Comment:
I realize in general these tests are somewhat placeholders because the JSON API isn't returning audit actions yet, but I was a bit confused by this comment, and I'd like to leave clear instructions for whoever picks up the next step and adds in the API part. 

At this point in the test, there will be one audit action, because you can't publish a new crate without publishing at least one version of that crate. Also, no audit actions have taken place between these lines and the lines that have the comment "make sure it has one audit action".

So I think this comment should say "make sure it has one audit action", the next line saying "do this as a full integration test once the api is in place" is fine to stay, and then remove the comments on lines 1609 and 1610 because they'll be redundant with the comments here.